src/Translator.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+namespace atk4\core;
+
+class Translator implements TranslatorInterface
+{
+    use ConfigTrait;
+
+    /**
+     * ISOCode of the main language.
+     *
+     * @var string
+     */
+    public $language;
+
+    /**
+     * ISOCode of the fallback language.
+     *
+     * @var string
+     */
+    public $fallback;
```

Comment:
Set default value to 'en', since that's what all the strings are in anyway, so it should be the fallback by default.